syft/frameworks/torch/rnn.py

```diff
@@ -1,11 +1,17 @@
+# TODO Workaround to cyclic import
+# HELP NEEDED HERE
+# This file is normally inside the nn/ directory. In the precision.py file
+# when we import nn, the nn/__init__.py gets called. Currently all other classes
+# (ie Conv2d, AvgPool2d, etc) behave well when imported in the nn/__init__.py
+# But due to the RNN Classes requiring the import of FixedPrecisionTensor (line 14)
+# and precision.py requiring the import of nn module, there is a problem.
```

Comment:
Do you think we could remove the import of FixedPrecisionTensor (not in this PR, of course)? I don't think RNN should deal with things at that low a level.